### ISSUE TITLE
Special 'nun' footnotes

### ISSUE THREAD

**Issue by mnjames** [mimbrown/render#44]
Our text has a certain kind of footnote which needs to be marked in the Greek text instead of the idiomatic translation.

My proposal would be to leave the actual footnote in SSV, but with a caller="-" instead of caller="+". This would hide the caller in the SSV text, while still having the information down at the bottom (with no ؂ symbol).

We would then need to hand-add some sort of code into the Greek, which SILE would need to read and add a superscript nun ن at the end of that Greek word (note this is a font change).

This would look like:
![noon](https://user-images.githubusercontent.com/30174022/54221836-0ced8300-4516-11e9-803c-b274051a3b05.png)
Note the nun in the footnote at the bottom is part of the footnote text; it's not automatically generated.

This could be put off post-31 March if need be.

**Comment by mnjames** [comment]
Attached are files showing what that data would look like in the future.
[Interlinear.xml.txt](https://github.com/mimbrown/render/files/4246972/Interlinear.xml.txt)
[Raw_Interlinear.xml.txt](https://github.com/mimbrown/render/files/4246973/Raw_Interlinear.xml.txt)
[Sinaiticus.usx.txt](https://github.com/mimbrown/render/files/4246974/Sinaiticus.usx.txt)
[SSV.usx.txt](https://github.com/mimbrown/render/files/4246975/SSV.usx.txt)
[SSV_Lit.usx.txt](https://github.com/mimbrown/render/files/4246976/SSV_Lit.usx.txt)
[Lexicon.xml.txt](https://github.com/mimbrown/render/files/4246985/Lexicon.xml.txt)

Specifically, the `Sinaiticus.usx` file includes a footnote at LUK 3:32 (and the same footnote will eventually be removed from `SSV.usx`).

Currently the footnote uses `+` as the generic marker, but we'd like a small noon to show up in the interlinear Greek and a corresponding noon in the bottom in line with the numbers. See a picture above to see how it should look.

**Comment by mnjames** [comment]
There is now an example of this in the ALEPH project at REV 1:11.

I have left the other footnote in the SSV for the time being, but once this is working we would delete all these sorts of footnotes in SSV and move them to ALEPH.

I'm still using the \zeng markers in the ALEPH project, though from a PT standpoint they're not necessary since the project itself is in Greek. We could delete those and make other modifications to the way these notes show up in ALEPH if necessary.

Instead of using the incremental markers (`\f + \ft...`) or the invisible marker (`\f - \ft...`), I've used a custom marker (`\f ن \ft...`). But that choice can also be changed--we just need some way for these notes to have "ن" as their caller instead of numbers.

**Comment by mimbrown** [comment]
I see the footnote, but it's marked as `\f + \ft`. Does that need to be changed to `ن`?

**Comment by mnjames** [comment]
Yes. I changed the marker after doing a send/receive, and forgot to do another one. I've done it now.

**Comment by mnjames** [comment]
This now works, but the placement isn't perfect (even allowing for verses that break across pages).

See: https://drive.google.com/open?id=1Giay0K2BS5crL1Ac53w_ysBtM4ZPXKe3

A footnote occurs on p. 37, where the last verse is 28.
The footnote symbol occurs on p.38, 6th line of Greek. It's in the correct location, in v.32.

I would expect the actual footnote to occur on p.38.
